css/main.css

```diff
@@ -0,0 +1,137 @@
+h1 {
+    color: #dedede;
+}
+
+p {
+    color: #eeeeee;
+}
+
+/* Navigation */
+
+.navbar {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.nav-link,
+.navbar-brand {
+    color: #dedede !important;
+    cursor: pointer !important;
+}
+
+.nav-link {
+    margin-right: 1em !important;
+}
+
+.nav-link:hover {
+    background: rgba(0, 0, 0, 0);
+    color: #af0404 !important;
+}
+
+.navbar-collapse {
+    justify-content: flex-end;
+}
+
+.navbar-toggler {
+    background: rgb(255, 0, 0, 0.7) !important;
+}
+
+.header {
+    background-image: url('../images/bkgtop.png');
+    background-attachment: fixed;
+    background-size: cover;
+    background-position: center;
+    width: 100%;
+    height: 850px;
+}
+
+.overlay {
+    position: absolute;
+    min-height: 100%;
+    min-width: 100%;
+    left: 0;
+    top: 0;
+}
+
+.description {
+    position: absolute;
+    top: 30%;
+    margin: auto;
+    padding: 2em;
+    width: 50%;
+}
+
+.description h1 {
+    color: #af0404;
+    font-size: 3rem;
+}
+
+.description p {
+    color: #dedede;
+    font-size: 1.2rem;
+    line-height: 1.5;
+}
+
+
+/* Community Section*/
+
+.community {
+    background-color: #af0404;
+    padding-bottom: 50px;
+}
+
+.text {
+    margin-top: 20%;
+}
+
+.buttons {
+    margin-top: 20%;
+    display: flex;
+    flex-direction: column;
+
+}
+
+/*Kizomba Section*/
+.kizomba {
+    background-color: #414141;
+    padding-bottom: 30px;
+    padding-top: 5px;
+}
+
+
+.img {
+    margin: auto !important;
```

Comment:
I noticed this is the desktop version only. Soon enough you'll do it responsive, right? It's a common best practice to do "mobile first development". You start by writing the code for mobile (simpler most of the time) and then add the media queries for desktop (add complexity). You can read more about [Start practicing mobile first development](https://getflywheel.com/layout/start-practicing-mobile-first-development/)

Reply:
I did not know that! Will read the article. Yes I was thinking to add @media after all done

Reply:
Okay, I assure you it's easier if you start the mobile version first and only after add the desktop version on top.